scripts/create-feature-annotations.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
For executable scripts, I recommend naming in kebab-case (which you're doing here) with no extension. The language shouldn't be part of the user interface for a CLI script.